src/Activity.js

```diff
@@ -1,3 +1,8 @@
+// if (typeof require !== 'undefined') {
+//   const UserRepository = require('../src/UserRepository.js');
+// }
+// userRepo = new UserRepository(userData);
```

Comment:
Running the index.html is what gives the Activity file access to the user Repo definition. That html file never runs in the tests, so userRepo is undefined